.evergreen/run-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+export GOPATH=$(dirname $(dirname $(dirname `pwd`)))
+export GOCACHE="$(pwd)/.cache"
+
+if [ "Windows_NT" = "$OS" ]; then
+    export GOPATH=$(cygpath -m $GOPATH)
+    export GOCACHE=$(cygpath -m $GOCACHE)
+
+    mkdir -p c:/libmongocrypt/include
+    mkdir -p c:/libmongocrypt/bin
+    curl https://s3.amazonaws.com/mciuploads/libmongocrypt/windows/latest_release/libmongocrypt.tar.gz --output libmongocrypt.tar.gz
+    tar -xvzf libmongocrypt.tar.gz
+    cp ./bin/mongocrypt.dll c:/libmongocrypt/bin
+    cp ./include/mongocrypt/*.h c:/libmongocrypt/include
+    export PATH=$PATH:/cygdrive/c/libmongocrypt/bin
+else
+    git clone https://github.com/mongodb/libmongocrypt
+    ./libmongocrypt/.evergreen/compile.sh
+fi
+
+export GOROOT="${GOLANG_LOCATION}"
```

Comment:
The `GOLANG_LOCATION` variable will be populated via the continuous matrix testing Evergreen project.